core/trino-main/src/main/java/io/trino/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -297,17 +298,21 @@ protected Node visitShowTables(ShowTables showTables, Void context)
         @Override
         protected Node visitShowGrants(ShowGrants showGrants, Void context)
         {
-            // TODO: make this method redirection aware
```

Comment:
@phd3  there is also a TODO comment here: https://github.com/trinodb/trino/blob/2e3f9d573cce7ba5ed38477f5272510205e1a374/core/trino-main/src/main/java/io/trino/metadata/MetadataManager.java#L2025-L2027

Am I correct in assuming that handling the `MetadataManager#listTablePrivileges` is not in the scope of this PR ?